assets/src/domain/eventEditor/services/apollo/mutations/datetimes/useOptimisticResponse.ts

```diff
@@ -55,28 +67,20 @@ const useOptimisticResponse = (): OptimisticResCb => {
 			case MutationType.Delete:
 				espressoDatetime = {
 					...espressoDatetime,
+					...datetime,
 					...filteredInput,
+					isTrashed: true,
+					cacheId: uuidv4(),
```

Comment:
shouldn't this entity already have a cacheId?
or is this done as a way to guarantee that it gets updated everywhere, such as removed from lists, etc?

Reply:
Whenever an entity gets updated, its cacheId needs to change to reflect UI changes.

Reply:
ahhh ok... so that always happens... kk that makes sense